src/utils.rs

```diff
@@ -498,6 +499,7 @@ impl Preferences {
 			use_hls: setting(&req, "use_hls"),
 			hide_hls_notification: setting(&req, "hide_hls_notification"),
 			autoplay_videos: setting(&req, "autoplay_videos"),
+			link_confirmation: setting(&req, "link_confirmation"),
```

Comment:
@sigaloid: FYI This is likely to cause conflict with your incoming config changes. I'll leave it to you to decide whether or not this PR should go in first or if we should wait until your config changes have been merged into master.